cmd/collector/app/root_span_handler_test.go

```diff
@@ -32,6 +32,10 @@ type mockAggregator struct {
 func (t *mockAggregator) RecordThroughput(service, operation string, samplerType model.SamplerType, probability float64) {
 	t.callCount.Add(1)
 }
+
+func (t *mockAggregator) HandleRootSpan(span *model.Span, logger *zap.Logger) {
```

Comment:
why is `cmd/collector/app/root_span_handler.go` needed at all if we move the functionality?